### ISSUE TITLE
Add coala

### ISSUE THREAD

**Issue by Monal5031** [anshumanv/evolutions-webapp#33]
##### This issue is about adding coala checks ifor codebase quality n repo.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by Monal5031** [comment]
@anshumanv Activate travis please.
